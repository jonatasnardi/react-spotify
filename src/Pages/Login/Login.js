import React, { Component } from 'react';
import { withRouter } from 'react-router-dom';
import "./Login.scss";
import { generateRandomString } from '../../Helpers/functions';

class Login extends Component {
  constructor(props) {
    super(props);
  }

  render() {
    return (
      <section className="login">
        <div className="container-buttons">

        </div>
        <h1 className="main-title">Clique no bot√£o abaixo para gerar seu acesso</h1>
        <a href='http://super-hobbies.herokuapp.com/'>
          <div class="css-button">
            <p class="css-button-text">GERAR ACESSO</p>
            <div class="css-button-inner">
              <div class="reset-skew">
                <p class="css-button-inner-text">GERAR ACESSO</p>
              </div>
            </div>
          </div>
        </a>
      </section>
    );
  }
}

export default withRouter(Login);