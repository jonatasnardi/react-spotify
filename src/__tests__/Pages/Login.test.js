import React from 'react';
import { shallow } from 'enzyme';
import Login from '../../Pages/Login/Login';

describe('Testing Login Component', () => {
  it('should render correctly', () => {
    const wrapper = shallow(
      <Login />
    );

    expect(wrapper).toMatchSnapshot();
  })
})