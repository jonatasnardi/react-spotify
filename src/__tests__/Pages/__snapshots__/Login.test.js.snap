// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Login Component should render correctly 1`] = `ShallowWrapper {}`;
